app/tests/unit/containers/Organisation/Organisations.test.tsx

```diff
@@ -31,31 +55,11 @@ describe('Organisations', () => {
     expect(r).toMatchSnapshot();
   });
   it("should render the user's requested organisations", async () => {
-    const query: Organisations_query = {
-      ' $refType': 'Organisations_query',
-      session: {
-        ciipUserBySub: {
-          id: '',
-          ciipUserOrganisationsByUserId: {
-            edges: [
-              {
-                node: {
-                  id: 'OrgId',
-                  ' $fragmentRefs': {UserOrganisation_userOrganisation: true}
-                }
-              }
-            ]
-          }
-        }
-      },
-      allOrganisations: {
-        edges: []
-      }
-    };
```

Comment:
Just moved the mock queries to the top of the page so I could re-use them in my new test